.gitignore

```diff
@@ -205,6 +205,9 @@ node_modules/
 /ajax/libs/slim.js/2.[0-5].*/
 /ajax/libs/europa/4.0.0-alpha*/
 
+
+/ajax/libs/angular-mocks/*-build*/
+
```

Comment:
I think maybe we don't need to add the more blank line. L208 and L210

Just add `/ajax/libs/angular-mocks/*-build*/` between L206`/ajax/libs/europa/4.0.0-alpha*/` and L207 (blank line)

What do you think?

Thank you.